### ISSUE TITLE
TMG support for Bentley\OpenPaths\EMME 24.00.00

### ISSUE THREAD

**Issue by malamillo** [TravelModellingGroup/TMGToolbox#238]
Hi all,
I believe the TMG toolbox needs some tweaking to be installed on OpenPaths EMME 2024.

I was able to build the TMG_Toolbox.mtbx by downloading the TMGToolbox-1.12.1420.zip and running the "Build Consolidated Toolbox.bat" but had to make the following changes:

From:
```
@echo off
if exist "%EMMEPATH%/Python27/python.exe" (
    "%EMMEPATH%/Python27/python.exe" "%~dp0\build_toolbox.py" -c
) else (
    "%EMMEPATH%/Python37/python.exe" "%~dp0\build_toolbox.py" -c
)
```

To:
```
@echo off
if exist "%EMMEPATH%/Python27/python.exe" (
    "%EMMEPATH%/Python27/python.exe" "%~dp0\build_toolbox.py" -c
) else (
    "%EMMEPATH%/Python311/python.exe" "%~dp0\build_toolbox.py" -c
)
```

A syntax warning was thrown back at me but did not crash the consolidation:
```
------------------------
 Build Toolbox Utility
------------------------

toolbox: E:\gghm_v5_code_main_20240626\TMGToolbox-1.12.1420\TMGToolbox\TMG_Toolbox.mtbx
source folder: E:\gghm_v5_code_main_20240626\TMGToolbox-1.12.1420\TMGToolbox\src
title: TMG Toolbox
namespace: tmg
consolidate: True

Loading toolbox structure
E:\gghm_v5_code_main_20240626\TMGToolbox-1.12.1420\TMGToolbox\src\common\TMG_tool_page_builder.py:40: SyntaxWarning: 'str' object is not callable; perhaps you missed a comma?
  _getExtraAttributeText = lambda exatt: "%s - %s - %s"(exatt.id, exatt.type, exatt.description)
Done! Found 161 elements.
```

The tools I tested seem to be functional so far.

Best regards.

**Comment by bccheung** [comment]
Hi @malamillo, the fix for this was addressed by #236. It is currently part of the dev branch

**Comment by malamillo** [comment]
Thanks @bccheung. I'll leave it to TMG to close the ticket then.
